src/extension.js

```diff
@@ -5,32 +5,32 @@ const extension = {
   options: {
     sync: new OptionSection(browser.storage.sync, {
       wrapNavigation: false,
+      hideOutline: false,
       autoSelectFirst: true,
-      nextKey: 'down, j',
-      previousKey: 'up, k',
-      navigatePreviousResultPage: 'left, h',
-      navigateNextResultPage: 'right, l',
-      navigateKey: 'return, space',
-      navigateNewTabBackgroundKey: 'ctrl+return, command+return, ctrl+space',
-      navigateNewTabKey:
-        'ctrl+shift+return, command+shift+return, ctrl+shift+space',
-      navigateSearchTab: 'a, s',
-      navigateImagesTab: 'i',
-      navigateVideosTab: 'v',
-      navigateMapsTab: 'm',
-      navigateNewsTab: 'n',
-      navigateShoppingTab: 'alt+s',
-      navigateBooksTab: 'b',
-      navigateFlightsTab: 'alt+l',
-      navigateFinancialTab: 'f',
-      focusSearchInput: '/, escape',
-      navigateShowAll: 'ctrl+shift+a',
-      navigateShowHour: 'ctrl+shift+h',
-      navigateShowDay: 'ctrl+shift+d',
-      navigateShowWeek: 'ctrl+shift+w',
-      navigateShowMonth: 'ctrl+shift+m',
-      navigateShowYear: 'ctrl+shift+y',
-      toggleSort: 'ctrl+shift+s'
+      nextKey: ['down', 'j'],
+      previousKey: ['up', 'k'],
+      navigatePreviousResultPage: ['left', 'h'],
+      navigateNextResultPage: ['right', 'l'],
+      navigateKey: ['return', 'space'],
+      navigateNewTabBackgroundKey: ['ctrl+return', 'command+return', 'ctrl+space'],
+      navigateNewTabKey: ['ctrl+shift+return', 'command+shift+return', 'ctrl+shift+space'],
+      navigateSearchTab: ['a', 's'],
+      navigateImagesTab: ['i'],
+      navigateVideosTab: ['v'],
+      navigateMapsTab: ['m'],
+      navigateNewsTab: ['n'],
+      navigateShoppingTab: ['alt+s'],
+      navigateBooksTab: ['b'],
+      navigateFlightsTab: ['alt+l'],
+      navigateFinancialTab: ['f'],
+      focusSearchInput: ['/', 'escape'],
+      navigateShowAll: ['z z', 'command+shift+a'],
```

Comment:
Shouldn't this and the others below include both `command` and `ctrl` shortcuts?